.kokoro/autogenerate.sh

```diff
@@ -62,9 +62,23 @@ fi
 # (We don't actually need to pack here, but it's simplest to always do so.)
 ./BuildGenerated.sh @tmp/ApisToGenerate.txt
 
+# For each new/updated Discovery file, add the Discovery file
+# and the generated code in a commit.
+for file in $(cat tmp/ApisToGenerate.txt | sed 's/\r//g')
+do
+  service=$(basename $file | sed 's/.json//g')
+  generated=$(dotnet run --project Src/Tools/BuildGeneratedArgumentTranslator -- Src/Generated $file)
+  git add $file $generated
```

Comment:
I think here we need to include the `service.json.original` if there's one, else it will be included on the "Delete obsolote discoveries".

Reply:
Ah yes. Will need to work out how to do that.

Reply:
It's possible that `git add ${file}* $generated` *might* work, but I'll need to test it.